test/preload/__snapshots__/prod.spec.ts.snap

```diff
@@ -177,6 +177,7 @@ exports[`preload tags > Preload Eager 2`] = `
     <html>
       <head>
         <link rel="stylesheet" type="text/css" href="/assets/static/renderer_Layout-031b266d.$HASH.css">
+        <link rel="preload" href="/assets/static/Sono-Light.$HASH.woff2" as="font" type="font/woff2" crossorigin>
         <link rel="preload" href="/assets/static/Sono-Light.$HASH.ttf" as="font" type="font/ttf" crossorigin>
```

Comment:
Development-like preloading, both fonts are included.
Debatable if we should keep it like this, but I guess it doesn't hurt since it was already like that.